### DISCUSSION TITLE
can i work with .NET core from CDR ?

### DISCUSSION THREAD

**Discussion by tsgiannis** [coder/code-server#2379] [Q&A] [answered]
Hello to everybody
I have this question and i tried to find an answer but in vain
I have installed CDR and is great running on my VPS.
I had a thought about working on one of my projects that is .NET Core based, so i installed a couple of extensions but in vain
When i try to run a project i get this message

> The user's home directory could not be determined. Set the 'DOTNET_CLI_HOME' environment variable to specify the directory to use
I did a search on Google but the info i get is a bit vague at least to me

> Try adding Environment=DOTNET_CLI_HOME=/temp to your Service declaration in your .service file. Example syntax:
> [Service]
> ...
> Environment=VARNAME=VARCONTENTS

Any ideas?

**Comment by tsgiannis** [comment, marked as answer]
Edit: just by reviewing the whole case i resolved it

**Reply by nhooyr** [reply]
Awesome!
